target/linux/realtek/rtl838x/config-5.15

```diff
@@ -1,5 +1,6 @@
 CONFIG_ARCH_32BIT_OFF_T=y
```

Comment:
@svanheule do we _rally_ need to have per chipset kernel/configs? Can we not do 1 unified kernel instead?

Reply:
No, we don't. Most of the config is shared, so we should be able to do something like the ath79 target.

Reply:
Excellent, so now we need a 'plan de campagne' :) We shouldn't do the kernel config part of this series, obviously. Personally, I think it would be much nicer to do the config merge 'first' to make this diff also smaller again.

I'd even argue, lets tie up some loose ends (MR's) and then do this migration. I also see a few patches in this MR that could be v5.10 contenders, so if we pull those out and do them individually, it makes this MR smaller (yet again) and ties up a few more loose ends. I'll look into the config and see what we can easily do there.